packages/sandbox/src/releases.ts

```diff
@@ -17,6 +17,23 @@ export const supportedReleases = [
   "2.4.1",
 ] as const;
 
+/** The latest major.min version **/
+export const supportedReleases = [
+  "4.0.2",
```

Comment:
Should this be 4.0.3 now? There's a couple bugfixes that would be nice to have.

Reply:
I've triggered the deploy for that, should appear on the next prod deploy and automatically get added

![Screen Shot 2020-10-23 at 11 56 22 AM](https://user-images.githubusercontent.com/49038/97025647-5c839580-1526-11eb-8c97-ecf2b86da1eb.png)